core/src/test/scala/com/github/johnynek/paiges/PaigesTest.scala

```diff
@@ -305,7 +302,7 @@ the spaces""")
     forAll { (a: Doc, b: Doc) =>
       import Doc._
 
-      if (deunioned(a).toSet.subsetOf(deunioned(b).toSet)) {
+      if (a.deunioned.toSet.subsetOf(b.deunioned.toSet)) {
         assert(a.isSubDocOf(b))
       }
```

Comment:
if we use a `SortedSet` and the Ordering on Doc, I think we can make a stronger statement:
```scala
if (a.isSubDoc(b)) { sortedSet(a.deunioned) subsetOf sortedSet(a.denunioned) }
```